src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,118 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++;
+ *   case 2: // violation, case 1 contains code but lacks break, return, throw or continue statement
+ *     i++;
+ *     break;
+ *   case 3: // OK
+ *     i++;
+ *     return;
+ *   case 4: // OK
+ *     i++;
+ *     throw new Exception();
+ *   case 5: // OK
+ *     i++;
+ *     continue;
```

Comment:
that wont compile

Reply:
You mean `throw new Exception()`? My code has
```
    public void foo() throws Exception{}
```
But I left it out to focus on the `switch`. Should I put it in?
If you mean `continue`, I wrapped a `while loop` outside the `switch` as commented above

Reply:
fixed